slides/body/lect-w07-classes.tex

```diff
@@ -614,7 +613,6 @@
 \item en automatisk \Emph{\texttt{toString}} som visar klassparametrarnas värde, 
 \item ett automatiskt \Emph{kompanjonsobjekt} med \Emph{fabriksmetod} så du slipper skriva \code{new},
 \item automatiska metoden \Emph{\texttt{copy}} för att skapa kopior med andra attributvärden, m.m...
-\item[] (Mer om detta i w06 \& w11, men är du nyfiken, kolla på uppgift 2d) på sid 261.)
```

Comment:
Keeping the w06 reference doesn't make much sense, but should we keep the reference to the future chapter+exercise? If so, how should we handle the page number now that the printed compendium doesn't have the same page numbers as the digital compendium?